### PR TITLE
Adds 'warble' emote for Skrell

### PR DESCRIPTION
**Pull request by iantine**
Adds 'warble' emote for Skrell

**What does this PR do:**
Adds *warble emote for Skrell, complete with sound effect. .ogg created from https://freesound.org/people/alienistcog/sounds/241978/

**Images of sprite/map changes:**
https://youtu.be/MZVfRKUsMxQ Working emote in question. Does not vary in pitch or change with age or gender of skrell, as in line with other species emotes.

**Changelog:**
:cl:
add: Skrell *warble emote
/:cl:

### CODE REVIEW

**Comment by TDSSS** [comment]
YES, GLORIOUS WARBLING

SPEEDMERGE

**Comment by Shadeykins** [comment]
Brilliant.

**Comment by taukausanake** [comment]
The fish bois are now frog bois. I don't know how to feel about this

**Comment by Bookdrizzt** [comment]
Holy cow, an actual reason to play Skrell.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -221,6 +227,13 @@
 			playsound(loc, 'sound/effects/voxrustle.ogg', 50, 0) //Credit to sound-ideas (freesfx.co.uk) for the sound.
 			m_type = 2
 
+		if("warble", "warbles")
+			var/M = handle_emote_param(param)
+
+			message = "<B>[src]</B> warbles[M ? " at [M]" : ""]."
+			playsound(loc, 'sound/effects/warble.ogg', 50, 0) //Gredit to alienistcog (freesound.org) for the sound.
```
Comment (Tayyyyyyy) [inline code/modules/mob/living/carbon/human/emote.dm]: This is probably a more license compliant way to cite the person. Also, "Credit" was mispelled
```suggestion
			playsound(loc, 'sound/effects/warble.ogg', 50, 0) // Copyright CC BY 3.0 alienistcog (freesound.org)
```

**Comment by KasparoVy** [comment]
Nice sound effect!

**Comment by KasparoVy** [comment]
Holy shit squid squad lets GO Skrell best species 2019!?

**Pull request merged**